### ISSUE TITLE
GEM naming convention needs change

### ISSUE THREAD

**Issue by JohnHadish** [SystemsGenetics/GEMmaker#201]
The naming convention for the final GEMs is like this:

```
{Machine_Name}.GEM.{output_format}.txt
```
So for example, a GEM ran on Kamiak with an output format of FPKM would be:
```
Kamiak.GEM.FPKM.txt
```

This provides very little useful information about what the file actually is. The machine name is probably not neccesary, as most users would more than likely be running everything on the same machine. I suggest removing the machine name option entirly from the `nextflow.config` as it seems like an unneccesary parameter at this point in GEMmaker.

We could use the parameter about the run, but I personally think this would result in more confusion than neccesary. The user should just rename the files in the output folder at the end of the run. I personally think that it would be best to have a constant name for the three potential files that are output rather than having users mess arround with parameters.

i.e.:
```
output:
   GEM.FPKM.txt
   GEM.TPM.txt
   GEM.Raw.txt
```

**Comment by JohnHadish** [comment]
I had no clue that this was the intent, as `machine_name` in this sense is not a term most people would recognize. It should be renamed to be `output_name`.

**Comment by spficklin** [comment]
It's actually been fully removed in the PR #205.  So, the prefix for all output files is just GEMmaker. (e.g. GEMmaker.TPM.txt).

**Comment by spficklin** [comment]
Fixed with PR #205. as per the comment above.
